R/init.R

```diff
@@ -94,17 +95,15 @@
 init <- function(data,
                  modules,
                  filter = teal_slices(),
-                 title = build_app_title(),
-                 header = tags$p(),
-                 footer = tags$p(),
-                 id = character(0),
-                 landing_popup = NULL) {
+                 title = NULL,
+                 header = NULL,
+                 footer = NULL,
```

Comment:
Please use `lifecycle::deprecated()` and `lifecycle::is_present()` instead of handling `NULL`

```suggestion
                 title = lifecycle::deprecated(),
                 header = lifecycle::deprecated(),
                 footer = lifecycle::deprecated(),
```